Add PNP info to PCI attachment of MFI, MALO, MWL, NFSMB, NGE, NTB_PLX, NLMSEC, NLMRSA, NFE, NVME drivers

lakhanshiva opened the pull request.

lakhanshiva committed:
Add PNP info to PCI attachment of MFI driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

bsdimp reviewed (commented):
looking good so far, though it's only one change and only a couple hours old...
I'll note that for some reason on all the pull requests you've added bktr and dpt. I've ignored them on the newer requests.

lakhanshiva committed:
Add PNP info to PCI attachments of MALO driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachment of MWL driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of NFSMB driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of NGE driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of NTB_PLX drivers

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachment og NLMSEC driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachment of NLMRSA driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachment of NFE driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of NVME driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of OCS driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

lakhanshiva committed:
Add PNP info to PCI attachments of OHCI driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/12
Url: https://reviews.freebsd.org/D16540

bsdimp commented on sys/dev/if_ndis/if_ndis_pci.c:
```diff
@@ -84,6 +84,10 @@ extern int ndis_detach		(device_t);
 extern int ndis_suspend		(device_t);
 extern int ndis_resume		(device_t);
 
+struct pci_device_table ndis_devs[] = {
```
ndis is a special driver. let's not make this change.

lakhanshiva commented:
Removed NDIS.
